### PR TITLE
Add documentation on alternative way of installing linkerd on dc/os

### PR DESCRIPTION
**Pull request by dadjeibaah**
The current `linkerd-dcos.json` install file used in linkerd examples can prevent the admin shutdown API feature in Linkerd from working. This documentation adds a note that describes a new way to set up a DC\OS linkerd configuration so admin shutdown API feature works in a DC\OS environment.

Signed-off-by: Dennis Adjei-Baah <dennis@buoyant.io>

### CODE REVIEW

File: dcos/linkerd-dcos-with-fetch.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "type": "DOCKER",
+    "volumes": [
+      {
+        "containerPath": "/io.buoyant/linkerd/1.3.6/config.yml",
+        "hostPath": "./config.yml",
+        "mode": "RW"
+      }
+    ],
+    "docker": {
+      "image": "buoyantio/linkerd:1.3.6",
```
Comment (adleong) [inline dcos/linkerd-dcos-with-fetch.json]: should we update this to 1.4.0?

Comment (dadjeibaah) [reply dcos/linkerd-dcos-with-fetch.json]: that's a good idea. I will change that

File: dcos/README.md [hunk 2]
```diff
@@ -14,6 +14,12 @@ dcos marathon app add webapp.json
 Note the `linkerd-dcos.json` files assume 4 nodes. Modify this to equal the
 total number of public+private nodes in your cluster.
 
+Note that the `linkerd-dcos.json` is one of many ways you can setup
+Linkerd to run in DC\OS. Unfortunately using `linkerd-dcos.json` might make it
+difficult to send operating system signals to Linkerd e.g. `SIGTERM` for
+graceful shutdown. An [alternative](/linkerd-dcos-with-fetch.json) is one way to
```
Comment (siggy) [inline dcos/README.md]: i think you want `[alternative](linkerd-dcos-with-fetch.json)` or `[alternative](/dcos/linkerd-dcos-with-fetch.json)`.

also you may want to note that fetch was introduced in 1.10, and that DC/OS <= 1.9, uses a top-level `uris` field.

File: dcos/README.md [hunk 3]
```diff
@@ -14,6 +14,12 @@ dcos marathon app add webapp.json
 Note the `linkerd-dcos.json` files assume 4 nodes. Modify this to equal the
 total number of public+private nodes in your cluster.
 
+Note that the `linkerd-dcos.json` is one of many ways you can setup
```
Comment (siggy) [inline dcos/README.md]: consider linking as:
[simple-proxy/linkerd-dcos.json](simple-proxy/linkerd-dcos.json)

File: dcos/linkerd-dcos-with-fetch.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      "executable": false,
+      "cache": false
+    }
+  ],
+  "container": {
+    "type": "DOCKER",
+    "volumes": [
+      {
+        "containerPath": "/io.buoyant/linkerd/1.3.6/config.yml",
+        "hostPath": "./config.yml",
```
Comment (siggy) [inline dcos/linkerd-dcos-with-fetch.json]: i think the `volumes` block is redundant with `fetch`?
looking at https://mesosphere.github.io/marathon/docs/application-basics.html#using-resources-in-applications, i think `fetch` should place `config.yml` in the current directory.

`volumes` seems like another interesting approach to getting a config file to linkerd, but perhaps we call  it out in a separate config?

File: dcos/README.md [hunk 5]
```diff
@@ -14,6 +14,12 @@ dcos marathon app add webapp.json
...
+Linkerd to run in DC\OS. Unfortunately using `linkerd-dcos.json` might make it
```
Comment (siggy) [inline dcos/README.md]: `DC/OS`

File: dcos/README.md [hunk 6]
```diff
@@ -14,6 +14,14 @@ dcos marathon app add webapp.json
 Note the `linkerd-dcos.json` files assume 4 nodes. Modify this to equal the
 total number of public+private nodes in your cluster.
 
+Note that the [simple-proxy/linkerd-dcos.json](simple-proxy/linkerd-dcos.json)
+is one of many ways you can setup Linkerd to run in DC/OS. Unfortunately using
+`linkerd-dcos.json` might make it difficult to send operating system signals to
+Linkerd e.g. `SIGTERM` for graceful shutdown. An
+[alternative](simple-proxy/linkerd-dcos-with-fetch.json)is one way to set up Linkerd so
```
Comment (siggy) [inline dcos/README.md]: `[alternative](simple-proxy/linkerd-dcos-with-fetch.json) is`

File: dcos/README.md [hunk 7]
```diff
@@ -14,6 +14,14 @@ dcos marathon app add webapp.json
...
+that it can catch os signals. This application definition uses the `fetch`
+API available in DC/OS 1.10. You can use a top-level `uri` list for DC/OS <= 1.9
```
Comment (siggy) [inline dcos/README.md]: `DC/OS <= 1.9.`

**Pull request merged**